src/components/site-form.tsx

```diff
@@ -505,24 +513,27 @@ export const SiteForm = ( {
 											</div>
 										) }
 
-										{ ! isWindows() && useCustomDomain && setEnableHttps && (
-											<div className="text-a8c-gray-50 text-xs mt-2">
-												{ __(
-													'You need to manually add the Studio root certificate authority to your keychain and trust it to enable HTTPS.'
-												) }{ ' ' }
-												<Button
-													variant="link"
-													onClick={ () => {
-														getIpcApi().openURL(
-															'https://developer.wordpress.com/docs/developer-tools/studio/ssl-in-studio/'
-														);
-													} }
-												>
-													{ __( 'Learn how' ) }
-													<span aria-label={ __( '(opens in a web browser)' ) }>&#8599;</span>
-												</Button>
-											</div>
-										) }
+										{ ! isCertificateTrusted &&
+											! isWindows() &&
+											useCustomDomain &&
+											setEnableHttps && (
+												<div className="text-a8c-gray-50 text-xs mt-2">
+													{ __(
+														'You need to manually add the Studio root certificate authority to your keychain and trust it to enable HTTPS.'
+													) }{ ' ' }
+													<Button
+														variant="link"
+														onClick={ () => {
+															getIpcApi().openURL(
+																'https://developer.wordpress.com/docs/developer-tools/studio/ssl-in-studio/'
+															);
+														} }
+													>
+														{ __( 'Learn how' ) }
+														<span aria-label={ __( '(opens in a web browser)' ) }>&#8599;</span>
```

Comment:
There should be a space between Learn how and the arrow. I also noticed that we are not using the arrow to express external link in other learn more links.

![Screenshot 2025-03-25 at 11 38 58](https://github.com/user-attachments/assets/80aa844e-341a-48c3-a459-01a2c1e5b509)

Figma design ( RToz6tIuQ7nlZrikBte4GU-fi-6924_4203 ):
![Screenshot 2025-03-25 at 11 44 47](https://github.com/user-attachments/assets/6ce94a1a-e732-4870-b791-239e92f6ceeb)